onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -692,6 +706,7 @@ def quantize_weight_per_channel(self, weight_name, weight_qType, channel_axis):
         if initializer is None:
             raise ValueError("{} is not an initializer", weight_name)
 
+        symmetric = 'Symmetric' not in self.extra_options or self.extra_options['Symmetric']
```

Comment:
could you add a member variable, like is_weight_symmetric? Then you don't need to load it multipe times